src/routes/ComparisonCreateRoute.js

```diff
@@ -166,6 +179,21 @@ class ComparisonCreateRoute extends React.Component {
 
         history.push(`/comparisons-erm/${comparisonId}${location.search}`);
         this.context.sendCallout({ message: <SafeHTMLMessage id="ui-erm-comparisons.comparison.created.success" values={{ name }} /> });
+      })
```

Comment:
Can we have all of this error handling done via Form Validation instead..similar to https://github.com/folio-org/stripes-erm-components/blob/master/lib/InternalContactsFieldArray/InternalContactsFieldArray.js#L108 what we have in `InternalContactsFieldArray`

Reply:
This is now dealt with by the requiredValidator